go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/viper v1.4.0
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/testify v1.4.0
```

Comment:
nit: could we not upgrade this in the same PR? I know it's fine but it makes the PR slightly bigger than it has to be.

Reply:
I needed to upgrade it to get access to the YAMLEq function. It's the only change that was made since `v1.3.0`